NEWS

```diff
@@ -12,6 +12,8 @@ Current git version
     following new values: 'off', 'hide_all', and 'hide_gaps'. Setting it to
     'hide_gaps' will only hide frame gaps when applicable, 'hide_all' and 'off'
     mirror the old behaviour with regards to 'true' and 'false'.
+  * New setting 'show_frame_decorations' that control, when frame decorations
+    are shown. This should be moved instead of the old 'always_show_frame'
```

Comment:
should be used?

Reply:
`control,` should be `controls`. And yes `moved` doesn't make sense. I think you meant `used`.